src/private/utils.js

```diff
@@ -62,13 +62,10 @@ const getOwnKeys = getOwnPropertySymbols
 
 
 export function getOwnPropertyDescriptors(obj) {
-  const descs = {};
-
-  getOwnKeys(obj).forEach(
-    key => (descs[key] = getOwnPropertyDescriptor(obj, key))
-  );
-
-  return descs;
+  return getOwnKeys(obj).map(key => [
```

Comment:
This is no longer equal to the upcoming `Object.getOwnPropertyDescriptors` that this intended to shim https://github.com/tc39/proposal-object-getownpropertydescriptors

That's a deal breaker this is approach. Totally cool with fixing this overall issue, but just need to keep the spec'd behavior of `getOwnPropertyDescriptors`